GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -619,8 +619,9 @@ private void CheckFileSystemSupportsRequiredFeatures(ITracer tracer, Enlistment
 
         private void CheckGitVersion(ITracer tracer, GVFSEnlistment enlistment, out string version)
         {
-            GitVersion gitVersion;
-            if (!GitProcess.TryGetVersion(out gitVersion, out string _))
+            string gitPath = GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath();

```

Comment:
The `enlistment` already has this path, there's no need to call this method again